src/build-info/src/lib.rs

```diff
@@ -30,16 +30,20 @@ pub struct BuildInfo {
 /// Intended for use in contexts where getting the correct build information is
 /// impossible or unnecessary, like in tests.
 pub const DUMMY_BUILD_INFO: BuildInfo = BuildInfo {
-    version: "",
-    sha: "",
+    version: "0.0.0+dummy",
+    sha: "0000000000000000000000000000000000000000",
     time: "",
     target_triple: "",
 };
 
 impl BuildInfo {
     /// Constructs a human-readable version string.
     pub fn human_version(&self) -> String {
-        format!("v{} ({})", self.version, &self.sha[..9])
+        if self == &DUMMY_BUILD_INFO {
+            "DUMMY_BUILD_INFO".to_string()
+        } else {
```

Comment:
The new contents of `DUMMY_BUILD_INFO` LGTM but we should be able to ditch the `if` now!

Reply:
:face_palm: